### DISCUSSION TITLE
Socket disconnect when react-native app goes to background

### DISCUSSION THREAD

**Discussion by zoobibackups** [socketio/socket.io#4346] [Q&A]
Socket disconnect when react-native app goes to background.

Even app is running is running in background with foreground service.

**How to keep socket connection live when app is running in background??**

**Comment by darrachequesne** [comment]
Hi! Socket.IO is not meant to be used in the background, as it will keep an open TCP connection to the server which will quickly drain the battery of the user.

Reference: https://socket.io/docs/v4/#what-socketio-is-not
